Good Coding Practices/src/js/scripts.js

```diff
@@ -1,13 +1,19 @@
 // piece object
 const piece = (function() {
   let el = null;
+  const initPos = null;
   const init = function(el) {
     this.el = el;
+    this.initPos = this.el.getBoundingClientRect();

```

Comment:
Either its an error or the new variable is not needed (you can't reassign a `const`)